mango-viz/bdgenomics/mango/pileup/utils.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+def parse_bed(json_input):
+    """ Converts a bed JSON string to a GA4GHFeature JSON String
```

Comment:
Would it maybe be easier to just pass the dataframe into this function and parse the rows directly? Then you wouldn't have to do all of the string replace work, and simply convert each row. You could use the function `build_json_from_bed` to map each row in the dataframe